### PR TITLE
Christos branch

### PR DESCRIPTION
**Pull request by cothon01**
Merging the initial asset folder structure.

### CODE REVIEW

File: Assets/Sprites/README.md.meta
Comment (Helcostr) [inline Assets/Sprites/README.md.meta:1]: Oh this is exeptionally good.

Or this need change...

**Review by Helcostr — CHANGES_REQUESTED** [review]
Test request

File: Assets/Sprites/README.md.meta
Comment (cothon01) [reply Assets/Sprites/README.md.meta:1]: Thanks :3

**Review by Helcostr — APPROVED** [review]
All good!

**Pull request merged**